src/components/ChatContent/ChatContent.module.css

```diff
@@ -25,7 +25,7 @@
   text-align: left;
   cursor: default;
   user-select: auto;
-  max-width: 100%;
```

Comment:
would flex-grow also work?

Reply:
Flex-grow is not the best option here, but the flex-shrink worked out!